front/app/components/UI/RHFFeedback/index.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import Error, { findErrorMessage, TFieldName } from 'components/UI/Error';
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+import {
+  Text,
+  Title,
+  Box,
+  colors,
+  Icon,
+} from '@citizenlab/cl2-component-library';
+import { scrollToElement } from 'utils/scroll';
+import CloseIconButton from '../CloseIconButton';
+import messages from './messages';
+
+type RHFFeedbackProps = {
+  successMessage?: string;
+} & InjectedIntlProps;
+
+const RHFFeedback = ({
```

Comment:
The component provides feedback to the user about whether the form has been submitted successfully or not.